src/main/kotlin/org/rust/ide/core/overrideImplement/generateUtil.kt

```diff
@@ -0,0 +1,112 @@
+package org.rust.ide.core.overrideImplement
```

Comment:
I think there isn't any convention yet? Kotlin itself is not consistent (example [three](https://github.com/JetBrains/kotlin/tree/master/idea/ide-common/src/org/jetbrains/kotlin/util)).